Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -1927,33 +1928,48 @@ def com_google_fonts_check_101(font_metadata,
   id = 'com.google.fonts/check/102',
   conditions = ['font_metadata']
 )
-def com_google_fonts_check_102(ttFont, font_metadata):
-  """Copyright notices match canonical pattern?"""
+def com_google_fonts_check_102(font_metadata):
+  """Copyright notices match canonical pattern in METADATA.pb"""
   import re
-  from fontbakery.utils import get_name_entry_strings
+  string = font_metadata.copyright
+  does_match = re.search(r'Copyright [0-9]{4} The .* Project Authors \([^\@]*\)',
```

Comment:
@felipesanches perhaps we could define our copyright pattern in the constants module?

Reply:
I tend to only split out things to the constants module if they are used by more than a single check. Otherwise I prefer to keep the code explicit, so that it is easier to grasp at one quick glance without opening multiple source files.